src/Components/CloudAccounts/NoCloudAccounts.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Button,
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const NoCloudAccounts = () => (
+  <EmptyState
+    variant="lg"
+    titleText="Cloud Accounts Available"
+    headingLevel="h4"
+    icon={CubesIcon}
+  >
+    <EmptyStateBody>
+      Cloud accounts appear here when they are connected through Integrations.
```

Comment:
Issue: Again, the word "Integrations" is supposed to be a link to that application